src/components/Tab/style.scss

```diff
@@ -40,11 +39,12 @@
     font-weight: 700;
 
     .tab__description {
-      color: $font-primary-color;
+      color: #8e949a;
     }
 
     &::before {
-      height: 5px;
+      height: 4px;
+      bottom: -12px;
```

Comment:
this line won't be needed if you add padding-bottom to the tab instead of margin

Reply:
good point! :D